deploy/cr.yaml

```diff
@@ -121,7 +128,7 @@ spec:
     image: perconalab/percona-server-mysql-operator:main-orchestrator
     imagePullPolicy: Always
 
-    size: 3
+    size: 0
```

Comment:
can we check that `clusterType: async` and do not start orchestrator even if user change it to 3

Reply:
we already do it: https://github.com/percona/percona-server-mysql-operator/pull/91/files#diff-91ee13a891cd7d20c59089733ada15406d28a5249194c7922b7e6e9fc1e96ac1R604-R606